src/platform/windows/input.cpp

```diff
@@ -1333,6 +1333,11 @@ namespace platf {
 
     // Allow either PS4/PS5 clickpad button or Xbox Series X share button to activate DS4 clickpad
     if (gamepad_state.buttonFlags & (TOUCHPAD_BUTTON | MISC_BUTTON)) buttons |= DS4_SPECIAL_BUTTON_TOUCHPAD;
+    
+    // Manual DS4 emulation: check if BACK button should also trigger DS4 touchpad click
+    if ((config::input.gamepad == "ps4"sv || config::input.gamepad == "ds4"sv) 
```

Comment:
Unless something changed recently, I don't think `ps4` is a valid option.

Reply:
Yeah I wasn't sure if that was still valid so I kept it consistent with https://github.com/ekchang/Sunshine/blob/5b052d7c2c059aa26fdf79cc87885af2f461f544/src/platform/windows/input.cpp#L1177-L1179

But I'll go ahead and update if we're confident ds4 is the only supported config here

Reply:
Thanks for the background... I think we should probably remove "ps4" from there as well. Would you take care of it in this PR?